wordcount.py

```diff
@@ -47,8 +47,31 @@
 # and builds and returns a word/count dict for it.
 # Then print_words() and print_top() can just call the utility function.
 
+
```

Comment:
Good job with this section, it looks very well done, although you could have used a list comprehension.